airbyte-integrations/connectors/source-mixpanel/unit_tests/utils.py

```diff
@@ -42,3 +50,14 @@ def read_incremental(stream_instance: Stream, stream_state: MutableMapping[str,
             stream_state = stream_instance.get_updated_state(stream_state, record)
             res.append(record)
     return res
+
+
+def init_stream(name="", config=None):
+    streams = SourceMixpanel(MagicMock(), config, MagicMock()).streams(config)
+    for stream in streams:
+        if stream.name == name:
+            # override Api Budget policies, as unit tests can fail due to full bucket
```

Comment:
This is interesting. It feels like this should be available in the CDK at some point. I've created [this issue](https://github.com/airbytehq/airbyte-internal-issues/issues/12302) and will share in the grooming to raise awareness on this. Thanks for tackling it for source-mixpanel